### ISSUE TITLE
 Make sure to call FirebaseApp.initializeApp(Context) first

### ISSUE THREAD

**Issue by Neustart** [arnesson/cordova-plugin-firebase#839]
First of all: I've read all related issues and I saw commit to #830 with a fix to this. 
Probably I having troubles with installing a new one version of the plugin.
This what I do:
```
ionic cordova plugin rm cordova-plugin-firebase
ionic cordova platform rm android
ionic cordova plugin add https://github.com/arnesson/cordova-plugin-firebase#1.1.4
ionic cordova platform add android@6.3.0
```
Version 1.1.4 should include a fix for this problem.
Any idea what am I do wrong?

**Comment by herrskytte** [comment]
I get this error also when using the new firebase plugin 2.0.0

**Comment by soumak77** [comment]
This should be fixed in v2.0.1.  Please comment if you are still seeing issues once upgrading.  You can also try downgrading to v1.0.5 if you are not yet willing to adopt v2.
